ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -37,13 +37,16 @@ function wait_until_pod_ready_status() {
   start_time=$(date +%s)
   while [ $(($(date +%s) - start_time)) -lt $timeout ]; do
      pod_status="$(oc get pod -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
+     daemon_status="$(oc get daemonset -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
      echo "$pod_status"
```

Comment:
```suggestion
     echo "Pod status: '$pod_status'"
```

Reply:
But that is the reason, no? To know exactly what happened... otherwise we wouldn't want to echo it at all.

Reply:
For that we have `echo "Waiting for Pods or Daemonset with label app='$pod_label' in namespace '$namespace' to be present..."` if either pod or daemnoset status is still unavailable.

Reply:
If we're not interested in this value to be printed everytime, then I vote to move these two `echo`s into the appropriate `if` statements. Otherwise, as you mentioned, the newline is printed empty many times. WDYT?